### DISCUSSION TITLE
Default password for live system

### DISCUSSION THREAD

**Discussion by powellnorma** [SpiralLinux/SpiralLinux-project#400] [Q&A]
The default user seems to be called `linux`. What is its default password? I want to login to the live-system over ssh. Thank you!

**Comment by geckolinux** [comment]
Hi there, there is no default password for `linux` .

**Comment by powellnorma** [comment]
Would it make sense to add a default password for such cases? I think one can't login to the ssh server otherwise.

**Reply by ghost** [reply]
Um, technically ssh not allowing blank passwd login is a good thing BUT allowing SpiralLinux to boot up sans a passwd is also good - but the two desires are incompatible, it would be best to boot up Spiral Live User then sudo su - ; passwd live and set it yourself...

No?  That way YOU control security.

:)  Just my 2 cents...

**Reply by powellnorma** [reply]
> but the two desires are incompatible

I don't think so, you can configure the tty service to not require a login. See for example: https://unix.stackexchange.com/a/42362

But I agree that not allowing password-less SSH-login over the network by default can also be seen as good think, because a live-user might not expect a "random" person (in same network) to be able to access his computer.

**Reply by geckolinux** [reply]
The recommendation would be to set a password for the `linux` user if you need to SSH into the live system.
